python/torch_mlir/__init__.py

```diff
@@ -177,10 +177,10 @@ def _canonicalize_args(example_args: _example_args_for_one_method):
         if not isinstance(example_args, Sequence):
             example_args = [example_args]
         for arg in example_args:
-            if not isinstance(arg, (TensorPlaceholder, torch.Tensor)):
-                raise Exception(f"Only Tensor's, TensorPlaceholder's, or sequences of "
-                                f"Tensor's and TensorPlaceholder's are supported as "
-                                f"example args for method inputs. "
+            if not isinstance(arg, (TensorPlaceholder, torch.Tensor, torch.SymInt)):
```

Comment:
Do you have an example of the use-case you're trying to handle with these changes?